scripts/edit_postman_json.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+jq --tab "del(.info._postman_id, .info._exporter_id)" EASI.postman_collection.json > tmp.postman && mv tmp.postman EASI.postman_collection.json
```

Comment:
Oh wow from the description I though this would be more in-depth. These are both internal to postman right?

Reply:
Yeah, and they pretty much get regen'd every time you export, and so I wanted to just add this to prevent conflicts when 2 branches update the collection!

We tried importing the collection without these and it seems to work just fine!

Reply:
OK cool, that would be my only concern is if Postman needed these values to import the collection.